alternatives.md

```diff
@@ -0,0 +1,113 @@
+# Babelfish vs Other Software
+
+<!-- TODO: https://github.com/oracle/opengrok/wiki/Comparison-with-Similar-Tools -->
+
+## [Kythe](https://kythe.io/)
+
+> The best way to view Kythe is as a “hub” for connecting tools for various languages, clients and build systems. By defining language-agnostic protocols and data formats for representing, accessing and querying source code information as data, Kythe allows language analysis and indexing to be run as services.
+
+> Some tools (e.g., static analyzers) already have expressive purpose-built internal representations for code. Kythe is not meant to be a universal replacement for such IRs — instead, our goal is to provide a way for such tools to capture “interesting subsets” of an analysis for sharing with other tools.
+
+Babelfish and Kythe share a goal of defining a common representation
+for concepts from different programming languages and provide a way to query it.
+Both provide a unified data format. And both are language-independent.
+
+The main difference is that Babelfish preserves all AST nodes including
+control flow and expressions, while Kythe focuses on class hierarchy,
+dependencies, etc.
+
+Also, Babelfish provides a unified IR across languages, which is defined
+as a non-goal of Kythe.
+
+Kythe requires to instrument language compilers and build systems while
+Babelfish uses native language parsers to get an AST, allowing to develop
+a new language driver in less time.
+
+Kythe also provides a way to process the whole project, while for now
+Babelfish is focused on processing individual files.
+
```

Comment:
I would add here that one of Kythe's main goals is to provide symbolic cross-referencing with as high accuracy as possible. Which is part of their mission to be the common format on which all developer tooling could be built. This is what leads them to need to process input from compilers and build systems. Babelfish today does not share that same goal, while in the future we might want to resolve references, but likely not to the extent that Kythe aims to do so. 

I would also mention that Kythe has worked with authors of compilers and build systems to be able to get the necessary information while Babelfish doesn't.

Reply:
Will mention build systems explicitly - it was assumed.

@eiso Also it would be nice to hear from you about the meeting with Kythe developers to know more about the goal of the project in more details.

Reply:
👍 Added a note about Kythe's main use case that drives the difference.